app/home/routes.py

```diff
@@ -91,4 +91,57 @@ def create_listing():
             img_file.save(file_path)
 
         return redirect(url_for('base_blueprint.route_default'))
-    return render_template('create_listing.html', form=form)
+    return render_template('create_property.html', form=form)
+
+
+@blueprint.route('/view-properties')
```

Comment:
You might want a login required decorator for this route as well

I see you are checking for permission level in templates 
Which is okay but the route must enforce that to start with

Reply:
As discussed, we leave the view properties open to public.